build/repo.props

```diff
@@ -66,6 +66,10 @@
                           "
                           Condition=" '$(MSBuildRuntimeType)' == 'Core' " />
 
+
+    <!-- Exclude the WPF samples for now because they use classic .csproj, which is not yet supported in our build. -->
+    <ProjectToExclude Include="$(RepositoryRoot)src\Identity\Identity\samples\NativeWPFClient\NativeWPFClient.csproj;" />
```

Comment:
This doesn't seem to work when I try to build the solution with dotnet build Identity.sln. Is that expected?

Reply:
yes, that's expected. You would have seen the same problem before unless you did `dotnet build -c DebugNoWPF Identity.sln`